src/django/contricleaner/lib/serializers/row_serializers/row_composite_serializer.py

```diff
@@ -22,6 +24,7 @@ def __init__(self, sector_cache: SectorCacheInterface):
             RowCountrySerializer(),
             RowRequiredFieldsSerializer(),
             RowEmptySerializer(),
+            RowFacilityTypeSerializer(),
```

Comment:
Please add it before RowEmptySerializer(),

Reply:
Done